src/main/scala/mesosphere/chaos/http/HttpConf.scala

```diff
@@ -40,4 +50,14 @@ trait HttpConf extends ScallopConf {
     // Default to the asset path in the jar
     case _                  => getClass.getClassLoader.getResource("assets")
   }
+
+  private lazy val httpCredentialsEnvName: String = "MESOSPHERE_HTTP_CREDENTIALS"
+  private lazy val httpCredentialsEnvValue: Option[String] = sys.env.get(httpCredentialsEnvName)
```

Comment:
Why `lazy val` instead of `def` here?  It doesn't look like they need to stick around after use.